include/OutboundEmail/OutboundEmail.php

```diff
@@ -249,30 +249,42 @@ function getUserMailers($user) {
 	function getUserMailerSettings(&$user, $mailer_id='', $ieId='') {
 		$mailer = '';
 
+
+		//  Sending emails uses personal instead of system outbound by default
+
+		if(!$ieId) {
+			if(!$user) {
+				global $current_user;
+				$user = $current_user;
+			}
+			$ieId = $user->getPreference('defaultIEAccount', 'Emails');
+		}
+
+
 		if(!empty($mailer_id)) {
 			$mailer = "AND id = '{$mailer_id}'";
 		} elseif(!empty($ieId)) {
-			$q = "SELECT stored_options FROM inbound_email WHERE id = '{$ieId}'";
-			$r = $this->db->query($q);
-			$a = $this->db->fetchByAssoc($r);
+			$query = "SELECT stored_options FROM inbound_email WHERE id = '{$ieId}'";
+			$results = $this->db->query($query);
+			$row = $this->db->fetchByAssoc($results);
 
-			if(!empty($a)) {
-				$opts = unserialize(base64_decode($a['stored_options']));
+			if(!empty($row)) {
+				$opts = unserialize(base64_decode($row['stored_options']));
 
 				if(isset($opts['outbound_email'])) {
 					$mailer = "AND id = '{$opts['outbound_email']}'";
 				}
 			}
 		}
 
-		$q = "SELECT id FROM outbound_email WHERE user_id = '{$user->id}' {$mailer}";
-		$r = $this->db->query($q);
-		$a = $this->db->fetchByAssoc($r);
+		$query = "SELECT * FROM outbound_email WHERE user_id = '{$user->id}' {$mailer} AND deleted = 0;";
```

Comment:
Oh, right question :) fortunately it doesn't change any behavior. I revert it back